code/modules/admin/verbs/randomverbs.dm

```diff
@@ -681,13 +681,6 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	var/confirm = alert(src, "You sure?", "Confirm", "Yes", "No")
 	if(confirm != "Yes") return
 
-	if(ticker.mode.name == "revolution" || ticker.mode.name == "AI malfunction" || ticker.mode.name == "confliction")

```

Comment:
@Jalleo i couldn't find anything about it in r4407, might have to ask the goons.